packages/core/src/node/env-variables/env-variables-server.ts

```diff
@@ -44,4 +52,36 @@ export class EnvVariablesServerImpl implements EnvVariablesServer {
         }
         return this.envs[key];
     }
+
+    getUserHomeFolderPathSync(): string {
+        return os.homedir();
+    }
+
+    async getUserHomeFolderPath(): Promise<string> {
+        return this.getUserHomeFolderPathSync();
+    }
+
+    getDataFolderNameSync(): string {
+        return THEIA_DATA_FOLDER;
+    }
+
+    async getDataFolderName(): Promise<string> {
+        return this.getDataFolderNameSync();
+    }
+
+    getUserDataFolderPathSync(): string {
```

Comment:
I don't think we need this method at all

Reply:
Actually, I also don't think the other sync variants are useful. The should at least not be public if they are not used anywhere.

Reply:
@svenefftinge We have to decide what we need, see https://github.com/eclipse-theia/theia/pull/6650#discussion_r358755213
I also see no big value in sync methods.